### PR TITLE
AdSense/Doubleclick remove delay request experiments

### PR DESCRIPTION
**Pull request by keithwrightbos**
Added in #16136 and being removed as unlaunchable

### CODE REVIEW

File: extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js [hunk 1]
```diff
@@ -84,23 +84,6 @@ export function resetSharedState() {
...
 }
 
-/**
- * Mapping of experiment id to viewport offsets by which ad request should
- * be delayed.
- * @type {!Object<string, number>}
- * @visibleForTesting
- */
-export const DELAY_REQUEST_EXP_BRANCHES = {
-  '21062224': true, // control, delay by renderOutsideViewport
```
Comment (bradfrizzell) [inline extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js]: Please remove all server side references to these experiments.

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js [hunk 2]
```diff
@@ -301,12 +288,6 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     return this.isAmpAdElement();
   }
 
-  /** @override */
-  delayAdRequestEnabled() {
-    return getExperimentBranch(this.win, RENDER_IDLE_DELAY_REQUEST_EXP) ==
```
Comment (bradfrizzell) [inline extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: Just want to doublecheck that the intent here is to fully delete, as opposed to in the adsense impl where it just returns true

File: extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js (hunk 1 above)
Comment (keithwrightbos) [reply extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js]: will do

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js (hunk 2 above)
Comment (keithwrightbos) [reply extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: confirmed

**Pull request merged**